doc/api/esm.md

```diff
@@ -46,6 +46,79 @@ property:
 
 * `url` {string} The absolute `file:` URL of the module.
 
+#### import()
+
+* {Function}
+
+The `import()` function is a `Function` used to dynamically import a module. Can import either CommonJS or ECMAscript Modules. Returns a Promise. Requires the following parameter:
```

Comment:
`import()` is actually NOT a  `Function` in ECMAScript, it's a function-like indeed, but it really is not. It does not implement any method of the `Function` prototype, so I think this needs to be rephrased (try to use `import.call` for example).

```suggestion
`import()` is a function-like keyword used to dynamically import a module - either CommonJS or ECMAscript Modules. It returns a Promise and requires the following parameter:
```